### PR TITLE
Adding unit tests for 'clients/hedera/mirror-node/model/transaction'.

### LINKED ISSUE #427: Unit tests for app/clients/hedera/mirror-node/model/transaction.go
- Implement unit tests for **app/clients/hedera/mirror-node/model/transaction.go**

### PR DESCRIPTION
**Pull request by Psykepro**
Signed-off-by: Nikolay Nedkov <nikolay.nedkov@limechain.tech>

**Detailed description**:
Adding unit tests for 'clients/hedera/mirror-node/model/transaction'.

**Which issue(s) this PR fixes**:
Fixes #427

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: app/process/handler/read-only/nft/fee/handler.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ package fee
 import (
 	"database/sql"
 	"github.com/hashgraph/hedera-sdk-go/v2"
-	mirror_node "github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera/mirror-node/model"
+	mirror_node "github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera/mirror-node/model/transaction"
```
Comment (vectorman1) [inline app/process/handler/read-only/nft/fee/handler.go:22]: mirrorNode

File: app/process/watcher/message/watcher_test.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ import (
 	"errors"
 	"fmt"
 	"github.com/hashgraph/hedera-sdk-go/v2"
-	"github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera/mirror-node/model"
+	message2 "github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera/mirror-node/model/message"
```
Comment (vectorman1) [inline app/process/watcher/message/watcher_test.go:23]: Fix import collision pls

File: test/mocks/service/readonly_service_mock.go [hunk 3]
```diff
@@ -17,7 +17,7 @@
 package service
 
 import (
-	mirror_node "github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera/mirror-node/model"
+	mirror_node "github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera/mirror-node/model/transaction"
```
Comment (vectorman1) [inline test/mocks/service/readonly_service_mock.go:20]: mirrorNode

**Review by svetlio8 — DISMISSED** [review]

**Review by vectorman1 — DISMISSED** [review]

**Review by vectorman1 — APPROVED** [review]

File: app/clients/hedera/mirror-node/model/transaction/transaction.go [hunk 4]
```diff
@@ -150,7 +151,7 @@ func (t Transaction) GetHBARTransfer(account string) (amount int64, isFound bool
 	return 0, false
 }
 
-// GetIncomingTransfer returns the transfer to an account in the following order:
+// GetIncomingTransfer returns the transfer to an transferAccountId in the following order:
```
Comment (failfmi) [inline app/clients/hedera/mirror-node/model/transaction/transaction.go:154]: nit: this has to be reverted

**Review by svetlio8 — DISMISSED** [review]

**Review by failfmi — APPROVED** [review]

**Pull request merged**